cpp/include/cudf/stream_compaction.hpp

```diff
@@ -138,6 +138,21 @@ cudf::table drop_duplicates(const cudf::table& input_table,
                             const cudf::table& key_columns,
                             const duplicate_keep_option keep,
                             const bool nulls_are_equal = true);
+
+/**
+ * @brief Count the unique elements in the column
+ *
+ * Given an input column, number of unique elements in this column is returned
+ * 
+ * @param[in] input_column The column whose unique elements will be counted.
+ * @param[in] dropna flag to ignore `NaN` & `null` in unique count if true.
+ * @param[in] nan_as_null flag to consider `NaN==null` if true.
+ *
+ * @return number of unique elements
+ */
+gdf_size_type unique_count(gdf_column const& input_column,
+                           bool const dropna,
```

Comment:
This needs to be clarified. If `dropna == true` and `nan_as_null == false`, what is the behavior? 

Also, this should have a more descriptive name, such as `ignore_nulls`.